scripts/monitoring/cron-send-etcd-status.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+'''
+   Command to send status of etcd to zabbix
+'''
+
+#This is not a module, but pylint thinks it is.  This is a command.
+#pylint: disable=invalid-name
+
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+import json
+import requests
+
+def main():
+    ''' Get data from etcd API
+    '''
+
+    API_HOST = 'https://localhost:4001/v2/stats/store'
+    SSL_CLIENT_CERT = '/etc/openshift/master/master.etcd-client.crt'
+    SSL_CLIENT_KEY = '/etc/openshift/master/master.etcd-client.key'
+
+    # Fetch the store statics from API
+    request = requests.get(API_HOST, cert=(SSL_CLIENT_CERT, SSL_CLIENT_KEY), verify=False)
```

Comment:
I would suggest adding another zabbix item called "openshift.master.etcd.ping" and when you're able to ping etcd, then send a 1. When you're not able to ping etcd, send a 0.

Currently, what happens here when etcd is down?

I would also place this in a try / catch, just to make sure.

That's the kind of logic that I do in the docker info gathering script.

See here:
https://github.com/openshift/openshift-tools/blob/master/docker/oso-rhel7-zagg-client/monitoring_scripts/cron-send-docker-metrics.py#L156-L180